addon/background.js

```diff
@@ -326,19 +326,31 @@ var MyQOnly = {
     if (bugzillaData.error) {
       throw new Error(`Bugzilla request failed: ${bugzillaData.error.message}`);
     }
-    let reviewTotal =
+
+    if (settings.allBugzillaFlags) {
+      let flagCounts = {};
+      bugzillaData.result.result.requestee.map(f => {
```

Comment:
Not sure we need to use a `map()` here, since we're not actually using the return value. This could probably be a forEach, or (preferable):

```js
let flagCounts = {};
for (let flag of bugzillaData.result.result.requestee) {
  if (!(f.type in flagCounts)) {
    flagCounts[f.type] = 0;
  }
  flagCounts[f.type]++;
}

return flagCounts;